### DISCUSSION TITLE
Add Export/Import Step to Transformers

### DISCUSSION THREAD

**Discussion by JeffBenedict** [nextgenhealthcare/connect#5275] [Feature/Enhancement Ideas]
I know that many folks use pure javascript in their transformers, but in come cases starting out using Mapper and Message Builder steps in a transformer are easier to wrap one's head around and use; but a caveat to using them is that they are not easily transferable from channel to channel.  It would be nice if there was an Export/Import Step(s) option.  This would make it almost as easy as doing a Copy/Paste for JavaScript.  I know you can Export/Import the entire Transformer, but what if I just want a single step from a transformer with multiple steps?  It would be nice to be able import a particular step from a transformer w/o having to append the entire thing and then remove the steps you don't want.

**Comment by pacmano1** [comment]
Add a feature reqeust in the "Issues" location here on GitHub.

**Reply by JeffBenedict** [reply]
I thought this was the right place to add Feature/Enhancement Ideas??   I didn't view it as an "Issue". but I'll do what you ask. 

![image](https://user-images.githubusercontent.com/106398018/177865046-aaaf4c3f-5d0a-4554-9a09-b81f17e84ebe.png)

Added: 
https://github.com/nextgenhealthcare/connect/issues/5276

**Reply by pacmano1** [reply]
Yeah, they need to clean up some of that stuff.

**Reply by tonygermano** [reply]
I think it's fine to add it here in discussions instead of issues. I don't know if one area is more likely to get looked at than the other.
